api/backend_test.go

```diff
@@ -1527,8 +1527,10 @@ func TestWalletConfigOnLoginAccount(t *testing.T) {
 	alchemyArbitrumSepoliaToken := "alchemy-arbitrum-sepolia-token"
 	alchemyOptimismMainnetToken := "alchemy-optimism-mainnet-token"
 	alchemyOptimismSepoliaToken := "alchemy-optimism-sepolia-token"
-	raribleMainnetAPIKey := "rarible-mainnet-api-key" // nolint: gosec
-	raribleTestnetAPIKey := "rarible-testnet-api-key" // nolint: gosec
+	alchemyBaseMainnetToken := "alchemy-base-mainnet-token" // nolint: gosec
```

Comment:
Alchemy is no longer doing this "one key per chain" nonsense, we should be able to use just a single key for all chains, no need to add more. I guess you can just take any from the map until we rework this? We should now probably have a single `alchemy-token` variable and remove the per-chain ones.